--- rails-3.2.19/lib/rails/generators/app_base.rb~	2014-08-31 21:37:50.000000000 +0200
+++ rails-3.2.19/lib/rails/generators/app_base.rb	2014-09-04 01:19:44.733043577 +0200
@@ -28,7 +28,7 @@
         class_option :skip_gemfile,       :type => :boolean, :default => false,
                                           :desc => "Don't create a Gemfile"
 
-        class_option :skip_bundle,        :type => :boolean, :default => false,
+        class_option :skip_bundle,        :type => :boolean, :default => true,
                                           :desc => "Don't run bundle install"
 
         class_option :skip_git,           :type => :boolean, :aliases => "-G", :default => false,
@@ -251,7 +251,7 @@
         # end-user gets the bundler commands called anyway, so no big deal.
         #
         # Thanks to James Tucker for the Gem tricks involved in this call.
-        print `"#{Gem.ruby}" -rubygems "#{Gem.bin_path('bundler', 'bundle')}" #{command}`
+        print `/usr/bin/bundle #{command}`
       end
 
       def run_bundle
